util/merkle/src/triedb.rs

```diff
@@ -105,6 +105,26 @@ impl<'db> TrieDB<'db> {
             None => Ok(None),
         }
     }
+
+    /// Check if every leaf of the trie exists
+    pub fn is_complete(&self) -> bool {
```

Comment:
By the way, what is the purpose of this method?

Reply:
As the comment above says, it checks if every single node of the trie actually exists in the DB.
Snapshot sync inserts the raw trie node directly into the database, so it's used for checking if the snapshot sync finished importing the entire trie.